### ISSUE TITLE
Sync requires admin; breaks Fluent (fix found)

### ISSUE THREAD

**Issue by lindeloev** [adirh3/Fluent-Search#925]
# TL;DR
Sync required admin rights I did not have. This made Fluent Search crash or become unresponsive. Deleting `%AppData%\Blast\Settings\Sync*.json` fixed it.

# Expected behavior
1. If needed, request login (as now).
2. If the user doesn't login, don't break.

# Detailed description
I successfully installed and used 1.0.22 on my work PC. I played around with a lot of settings and I guess Sync was one of them. Everything worked fine. After a PC reboot (which also relaunched Fluent Search), it opened the browser with the following (I cropped out my work logo and email address):

![image](https://github.com/adirh3/Fluent-Search/assets/1439417/20537dd4-7070-47e4-a898-97b3d2315ac8)

I don't have those rights, so I closed the dialogue. Now the Fluent Search UI didn't display at all, neither using shortcut keys or starting Fluent Search from the start menu, both of which used to focus FS previously. 

## Debugging:
But the process was still running. Ending the process and restarting FS caused the login/rights-dialogue to appear again. I tried installing 1.0.24 and the behavior was the same. 

I tried deleting all settings in `%AppData%\Blast\Settings\` and now Fluent Search launched fine. Pasting in all the previous settings cased a different behavior: Fluent Search would actually display but it would just be loading indefinitely (i.e., being useless), not indexing any apps, loading any settings, etc:

![image](https://github.com/adirh3/Fluent-Search/assets/1439417/c48cc876-06ee-4a98-ba44-804e4d864fa0)

After a bit of informed guess-trial-and-error, deleting `%AppData%\Blast\Settings\Sync*.json` caused everything to go back to normal.

# System info
Windows 10 22H2
Installed via nightly.

Thanks for an *>>awesome<<* productivity tool, which I recommend everywhere I go!

**Comment by adirh3** [comment]
Thanks for the detailed issue @lindeloev.
Will fix it.
